sync-timeoffs/SyncTimeOffs.js

```diff
@@ -194,6 +194,153 @@ async function syncTimeOffs() {
     if (firstError) {
         throw firstError;
     }
+
+    return true;
+}
+
+
+/** Example how to utilize the unsyncTimeOffs_() function.
+ *
+ */
+async function recoverEventsExample() {
+    const updateDateMin = new Date('Oct 21, 2024, 6:34:35 PM');
+    const updateDateMax = new Date('Oct 21, 2024, 10:50:50 PM');
+    const specs = [
+        ['example@giantswarm.io', "Example P32  ⇵"],
+        ['foobar@giantswarm.io', "Out - Concert ⇵"],
+        ['baz@giantswarm.io', "Vacation:  ⇵"],
+        // ...
+    ];
+
+    for (const [email, title] of specs) {
+        if (!await recoverEvents_(title, email, updateDateMin, updateDateMax)) {
+            break;
+        }
+    }
+}
+
+/** Utility function to recover accidentally cancelled events.
```

Comment:
Why do we need this one?

Reply:
To cleanup accidentally deleted events: https://gigantic.slack.com/archives/C05KK8S6CE8/p1729632739678839?thread_ts=1729535149.943309&cid=C05KK8S6CE8

Reply:
Similar operations were required several times after bugs caused confusion. It can come in handy in future, thus I left it included. Think of it as an undelete tool.